src/TSTransformer/util/types.ts

```diff
@@ -78,14 +78,8 @@ export function isDefinedType(type: ts.Type) {
 	);
 }
 
-export function isAnyType(type: ts.Type) {
-	return (
-		!!(type.flags & ts.TypeFlags.Any) &&
-		!(
-			ts.getObjectFlags(type) &
-			(ts.ObjectFlags.CouldContainTypeVariables | ts.ObjectFlags.CouldContainTypeVariablesComputed)
-		)
-	);
+export function isAnyType(state: TransformState): TypeCheck {
+	return type => type === state.typeChecker.getAnyType();
```

Comment:
Change unrelated to PR, maybe make a separate commit for future history purposes?